src/GafferImage/FormatQuery.cpp

```diff
@@ -67,6 +67,24 @@ setV2PlugComponentValue( T const& parent, Gaffer::NumericPlug< typename T::Value
 
 }
 
+// TODO - merge with ViewScope in ImageStats, decide where it should live
+class ViewScope : public Gaffer::Context::EditableScope
+{
+public:
+	ViewScope( const Gaffer::Context *context, std::string &view, const std::vector< std::string > &views )
+		: EditableScope( context )
+	{
+		if( view.size() && views.size() )
+		{
+			if( std::find( views.begin(), views.end(), view ) == views.end() )
+			{
+				throw IECore::Exception( "ImageStats : View does not exist \"" + view + "\"" );
+			}
+			set( GafferImage::ImagePlug::viewNameContextName, &view );
+		}
```

Comment:
One other potential upside of the `default` view could be that `view == ""` in FormatQuery could mean "don't modify the context", so that by default FormatQuery operates on the view provided by the downstream context. Or do we think folks should need to enter `${image:view}` explicitly for that? We should review the other query nodes for precedent, but I _think_ the scene ones work this way...

Reply:
Leaving this open, since I think it warrants a bit of discussion. At the very least, I think we'd need to improve the empty value in the `FormatQuery.view` UI?

Reply:
Hmm, yes, I had forgotten about the custom UI when I took your suggestion for "view == "" means don't modify context".  Should I add some sort of metadata like `viewPlugValueWidget:allowUseCurrentContext`, which adds a menu option called "Use Current Context", with a value of ""?

Reply:
Yeah, that sounds good.